docs/scripting/scripts.lib/ok.mdx

```diff
@@ -19,13 +19,15 @@ No known parameters.
 ### Return
 
 ```js
-Success
+Success;
```

Comment:
```suggestion
Success
```

I assume that this block is attempting to show what the CLI would output if this value was returned. Can you make that more clear in the text outside of it?